common/src/main/java/com/movie/recommender/common/model/movie/MovieAllData.java

```diff
@@ -0,0 +1,10 @@
+package com.movie.recommender.common.model.movie;
+import lombok.Data;
+
+
+@Data
+public class MovieAllData {
+    private Movie movie;
+    private MovieCredits movieCredits;
```

Comment:
instead of using `MovieCredits` and `MovieKeywords` store lists of `CastMember`, `CrewMember` and `Keyword`